### ISSUE TITLE
[Question]Embedding Generate Problem

### ISSUE THREAD

**Issue by SupUnicorn** [huggingface/pytorch-transformers#403]
When i use 'bert-base-uncase' model to get english embeddings,I can get accurate and unchangeable tensor.But when I do same things on 'bert-base-chinese',I input same sequence every times,but I get different tensors.My input senquece like ''[CLS]+sequence+[SEP]"

**Comment by KavyaGujjala** [comment]
Hi,

How did you get the english embeddings using the bert-base-uncase model.

I have fine tuned the model on domain specific corpus. I am trying to get embeddings now using the pytorch_model.bin I got after fine tuning.

Any idea on how to do this?

**Comment by thomwolf** [comment]
Are your models in evaluate mode (`model.eval()`)?
Usually reproductibility issues comes from forgetting to disable the DropOut modules using `model.eval()`.

**Comment by SupUnicorn** [comment]
> Are your models in evaluation mode (`model.eval()`)?
> Usually reproductibility issues comes from forgetting to disable the DropOut modules using `model.eval()`.

Oh,it's really helpful.My rediculous mistake.Thank you for your answer!!!

**Comment by rsc90** [comment]
@SupUnicorn  Following the above comment, could you please let me know , during the evaluation while reproducing the results using a finetuned model, how to disable the DropOut modules ?
